test/unit-tests.test.ts

```diff
@@ -19,23 +165,160 @@ beforeAll(() => {
 })
 
 afterAll(() => {
-  
+
 })
 
 describe('MultiIndexMap: ', () => {
-  test.todo("Initialization: ", () => {
-    
+  describe('Initialization: ', () => {
+    it("should map items  using indexName[3] ", () => {
+      let multiIndexMap = new MultiIndexMap(['id', 'email', 'phone'], testdata);
+      expect(multiIndexMap.has('phone', '1-360-575-0309 x33137')).toBe(true);
+    }
+    );
+
+    it("should map items  using indexName[2] ", () => {
+      let multiIndexMap = new MultiIndexMap(['id', 'email', 'phone'], testdata);
+      expect(multiIndexMap.has('email', "Anahi.Hoppe98@hotmail.com")).toBe(true);
+    }
+    );
+
+
+    it(" should map items  with two indexName as params ", () => {
+      const multiIndexMap = new MultiIndexMap(['id', 'phone'], testdata);
+      expect(multiIndexMap.has('phone', "424.841.4682 x15799")).toBe(true);
+    }
+    );
+
+    it(" should map items with only one indexName as param ", () => {
+      const multiIndexMap = new MultiIndexMap(['email'], testdata);
+      expect(multiIndexMap.has('email', "Anahi.Hoppe98@hotmail.com")).toBe(true);
+    }
+    );
+
   });
-  test.todo("get:", () => {
+
+
+  describe('get: ', () => {
+
+    it(" should find , get and match data using indexName[0] as indexName ", () => {
+      let multiIndexMap = new MultiIndexMap(['id', 'email', 'phone'], testdata);
+      let expectedData = {
+        "id": "ecc90d26-cea9-4f1d-bb6c-75be45451b78",
+        "email": "Anahi.Hoppe98@hotmail.com",
+        "phone": "1-360-575-0309 x33137",
+        "name": "Luz Hammes Jr.",
+        "address": "315 Kovacek Square",
+        "about": "Quibusdam et tenetur et eveniet laborum error.",
+        "age": 87329
+      }
```

Comment:
the same expected Data is used thrice in this , you can put this expected data in outer scope , in the describe and reuse it in all the three places